src/components/FieldApi.js

```diff
@@ -7,32 +7,41 @@ class FieldApi extends React.Component {
   render () {
     const { render, component, children, field, ...rest } = this.props
 
-    const {
-      formApi,
-    } = this.context
+    const { formApi } = this.context
 
     // Get the full field name
     const fullField = formApi.getFullField(field)
 
     // Get the node of that field
     const node = formApi.getNodeByField(fullField)
 
-    // Get the field state and api
+    // Get the field api
     const fieldApi = node ? node.api : {}
-    const fieldState = node ? node.getState() : {}
```

Comment:
I had already fixed this issue in master, i think the issue was that before nested fields was not implimenting the getState funciton and in master i think its added